Twitch/serve.py

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env python
```

Comment:
Overall: looks good. You added functionality, and did a lot of commenting. One comment on the commenting - Be sure that you're thinking about how much commenting makes sense. A lot of the comments are great, but some of them aren't as necessary.